data/obs-packaging/cc-oci-runtime/README.md

```diff
@@ -0,0 +1,18 @@
+#cc-oci-runtime
+
+This directory contains the sources to update rpm specfiles and debian source
+control files to generate ``cc-oci-runtime`` packages for Fedora and Ubuntu.
+
+There is a script to generate specs and dsc automatically, to be pushed to
+Open Build Service (http://openbuildservice.org/) and generate packages for you.
```

Comment:
Those files can be pushed to [Open Build Service] ( http://openbuildservice.org/) to generate the packages.